### ISSUE TITLE
AudioClip and AnimationClip have the same prefix `A_`

### ISSUE THREAD

**Issue by fguillen** [justinwasilenko/Unity-Style-Guide#1]
This makes difficult to filter these types individually

- https://github.com/justinwasilenko/Unity-Style-Guide#422-animations
- https://github.com/justinwasilenko/Unity-Style-Guide#429-audio

**Comment by justinwasilenko** [comment]
That's true, and also in fact not what I have been using. Have been working on some more changes to this as I build my new project.
For animations, I have been using the suffix _anim
